examples/hello/bpTimeWriter/helloBPTimeWriter.py

```diff
@@ -19,27 +19,34 @@
 # User data
 myArray = np.array([0, 1, 2, 3, 4, 5, 6, 7, 8, 9])
 nx = myArray.size
+time = np.array([0.0])
 
 # ADIOS
 adios = adios2.ADIOS(comm, adios2.DebugON)
 
 # IO
 bpIO = adios.DeclareIO("BPN2N")
+bpIO.SetEngine('bp3')
+
+fileID = bpIO.AddTransport('File', {'Library': 'fstream'})
```

Comment:
Also, is this a must? I guess this example is written to showcase a simple generic workflow. It might have digressed a little by specifying an outdated engine, as well as this particular transport.

Reply:
Fair.
It also works if you delete these two lines. Would you prefer I do that?

Reply:
Yeah, that would be great. Thanks!

Reply:
done